wp-content/plugins/pdf-creator-lite/functions.php

```diff
@@ -8,7 +8,7 @@ function SSAPDF_addAdminPage ()
 {
 	$page_title = 'PDF Creator';
 	$menu_title = 'PDF Creator';
-	$capability = 'edit_pages';
+	$capability = 'manage_options';
```

Comment:
What's this change about? It looks like only super administrators can create PDFs with this capability change